docs/working-with-projects/working-with-images/create-tag-retention-rules.md

```diff
@@ -7,7 +7,7 @@ A repository can rapidly accumulate a large number of artifacts, many of which m
 
 ## How Tag Retention Rules Work
 
-You define tag retention rules on repositories, not on projects. This allows for greater granularity when defining your retention rules. As the name suggests, when you define a retention rule for a repository, you are identifying which tags to retain. You do not define rules to explicitly remove tags. Rather, when you set a rule, any tags in a repository that are not identified as being eligible for retention are discarded. 
+You define tag retention rules on repositories, not on projects. This allows for greater granularity when defining your retention rules. As the name suggests, when you define a retention rule for a repository, you are identifying which tags to retain. You do not define rules to explicitly remove tags. Rather, when you set a rule, any tags in a repository that are not identified as being eligible for retention are discarded. `OR` algorithm is used between rules.
```

Comment:
@bitsf I would say either "An `OR` algorithm is used between rules" or "The `OR` algorithm is used between rules".